api/src/main/java/org/openmrs/module/serialization/xstream/mapper/CGLibMapper.java

```diff
@@ -15,6 +15,7 @@
 public class CGLibMapper extends MapperWrapper {
 	
 	public static final String marker = new String("EnhancerByCGLIB");
+	public static final String hibernateProxy = new String("$HibernateProxy$");

```

Comment:
Take a look at our naming convention for constants: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-NamingConventions

Reply:
@dkayiwa Ok
It should be fine now

Reply:
We also encourage including a link to the ticket to help the reviewer easily get to it: https://wiki.openmrs.org/display/docs/Pull+Request+Tips